ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx

```diff
@@ -0,0 +1,155 @@
+        E.fold(constNull, decoded => {
+          if (DateFromString.is(decoded)) {
+            const formattedDate = localeDateFormat(decoded, "%d/%m/%Y");
+            return <ClaimLabel>{formattedDate}</ClaimLabel>;
+          } else if (EvidenceClaim.is(decoded)) {
+            return <ClaimLabel>{JSON.stringify(decoded)}</ClaimLabel>;
+          } else if (ImageClaim.is(decoded)) {
+            return (
+              <Image
+                source={{ uri: decoded }}
+                style={{
+                  width: "23.45%",
+                  aspectRatio: 3 / 4
+                }}
+                resizeMode="contain"
+                accessibilityIgnoresInvertColors
+              />
+            );
+          } else if (DrivingPrivilegesClaim.is(decoded)) {
+            const privileges = decoded.map(p => p.driving_privilege).join(", ");
+            return <ClaimLabel>{privileges}</ClaimLabel>;
+          } else if (PlaceOfBirthClaim.is(decoded)) {
+            return <ClaimLabel>{decoded.locality}</ClaimLabel>;
+          } else {
+            return <ClaimLabel>{decoded}</ClaimLabel>;
+          }
+        })
+      ),
+    [claim]
+  );
+
+  if (!claimContent) {
+    return null;
+  }
+
+  return (
+    <CardClaimContainer testID={testID} position={position}>
+      {claimContent}
+    </CardClaimContainer>
+  );
+};
+
+export type CardClaimRendererProps<T> = {
+  // A claim that will be used to render a component
+  claim: ParsedCredential[number];
+  // Function that check that the proviced claim is of the correct type
+  is: (value: unknown) => value is T;
+  // Function that renders a component with the decoded provided claim
+  component: (decoded: T) => React.ReactElement | Iterable<React.ReactElement>;
+};
+
+/**
+ * Allows to render a claim if it satisfies the provided `is` function
+ * @returns The component from the props if value if correctly decoded, otherwise it returns null
+ */
+const CardClaimRenderer = <T,>({
+  claim,
+  is,
+  component
+}: CardClaimRendererProps<T>) =>
+  pipe(
+    claim.value,
+    ClaimValue.decode,
+    O.fromEither,
+    O.filter(is),
+    O.fold(constNull, component)
+  );
+
+// O.filter(is), O.fold(constNull, component)
+
+export type CardClaimContainerProps = WithTestID<{
+  position?: AbsoluteClaimPosition;
+  children?: React.ReactNode;
+}>;
+
+/**
+ * Component that allows to position a claim using "x" and "y" absolute values
+ * This components takes all the available space inside the parent component, then
+ * the "x" and "y" values are transformed in a relative position using the "padding".
+ */
+const CardClaimContainer = ({
+  position,
+  children,
+  testID
+}: CardClaimContainerProps) => (
+  <View
+    testID={testID}
+    style={[
+      styles.container,
+      {
+        left: position?.x,
+        top: position?.y
+      }
+    ]}
+  >
+    {children}
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    position: "absolute",
+    width: "100%",
```

Comment:
Setting each claim's width and height to 100% uses the parent's width and height, which is the entire card: you can see the result by applying a background color and a border.

Although it's not immediately visible to the user, I think it would better if claims occupy only the space they actually need, without unnecessary overflow. 

|The problem|How it should be|
|-------------|----------------|
|<img src="https://github.com/user-attachments/assets/b5567563-9270-4c9b-8b15-590e112eb752" width="240"/>|<img src="https://github.com/user-attachments/assets/741ef223-41d9-42f5-9aeb-e40f9f78121f" width="240" />|

What do you think?

Reply:
Yes, this was done on purpose. It is a way to use padding as a coordinate system, allowing claims to be positioned using % values. Without 100% in both width and height, this would not be possible.

Reply:
Sorry, I've misunderstood your comment. Now I see that the 100% width and height has also been included in the single claim, and in fact it is not okay. I'll fix it in the next commit. Thank you!

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6112/commits/0ef0186b41f2d8d3aba40777e1752fb9ded626c3